docs/mnist_example/README.md

```diff
@@ -196,19 +201,17 @@ int main() {
   int64_t rank = 4;
   int64_t shape[] = {1, 1, 28, 28};
 
-  // Create a tensor using omTensorCreateWithOwnership (returns a pointer to OMTensor).
+  // Create a tensor using omTensorCreateWithOwnership (returns pointers to OMTensor). 
```

Comment:
I think I missed one ... will update it to `omTensorCreate`. Thanks

Reply:
I just removed the `omTensorList` statement because that subsection is talking about freeing memory using the `owning` flag and that no longer pertains to `omTensorList` since it is always owning. Let me know if you are okay with me removing it.

Reply:
Updates